azurerm/resource_arm_servicebus_topic_authorization_rule.go

```diff
@@ -61,27 +69,41 @@ func resourceArmServiceBusTopicAuthorizationRuleCreateUpdate(d *schema.ResourceD
 	topicName := d.Get("topic_name").(string)
 	resourceGroup := d.Get("resource_group_name").(string)
 
+	if d.IsNewResource() {
+		// first check if there's one in this subscription requiring import
+		resp, err := client.GetAuthorizationRule(ctx, resourceGroup, namespaceName, topicName, name)
+		if err != nil {
+			if !utils.ResponseWasNotFound(resp.Response) {
+				return fmt.Errorf("Error checking for the existence of Service Bus Topic Rule %q (Resource Group %q / Namespace %q): %+v", name, resourceGroup, namespaceName, err)
```

Comment:
>				return fmt.Errorf(" [](start = 0, length = 23)

Move this error message as template function in `errors` package?

Reply:
this varies wildly by resource, so this doesn't make sense to refactor out